dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -736,9 +739,29 @@ public void testBasicCollections() {
         PublishRequest publishRequest = SwaggerUtility.createPublishRequest(true);
         containersApi.publish(entryId, publishRequest);
 
+        // Able to retrieve the collection and organization an entry is part of, even if there aren't any
+        EntriesApi entriesApi = new EntriesApi(webClientUser2);
+        List<CollectionOrganization> collectionOrganizations = entriesApi.entryCollections(entryId);
+        Assert.assertEquals(0, collectionOrganizations.size());
+
         // Add tool to collection
         organizationsApi.addEntryToCollection(organization.getId(), collectionId, entryId);
 
+        // Able to retrieve the collection and organization an entry is part of
+        collectionOrganizations = entriesApi.entryCollections(entryId);
+        Assert.assertEquals(1, collectionOrganizations.size());
+        CollectionOrganization collectionOrganization = collectionOrganizations.get(0);
+        Assert.assertEquals(organization.getId(), collectionOrganization.getOrganization().getId());
+        Assert.assertEquals(collection.getId(), collectionOrganization.getCollection().getId());
+
+        // Unable to retrieve the collection and organization of an entry that does not exist
+        try {
```

Comment:
In the future, this would be simpler as just a new test in the class since this doesn't really rely on anything in particular 
(makes it easier to debug)